### PR TITLE
mpvScripts.mpvacious: init at 0.12

### PR DESCRIPTION
**Pull request by kmicklas**
###### Motivation for this change

This is a useful MPV script for language learners.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/mpv/scripts/mpvacious.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    mkdir -p $out/share/mpv/scripts
+    cp subs2srs.lua $out/share/mpv/scripts
+  '';
+
+  passthru.scriptName = "subs2srs.lua";
+
+  meta = with lib; {
+    description = "Adds mpv keybindings to create Anki cards from movies and TV shows";
+    homepage = "https://github.com/Ajatt-Tools/mpvacious";
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/video/mpv/scripts/mpvacious.nix:29]: Plus or Only?

File: pkgs/applications/video/mpv/scripts/mpvacious.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchzip, curl, xclip }:
+
+stdenv.mkDerivation rec {
+  pname = "mpvacious";
+  version = "v0.12";
+  src = fetchzip {
+    url = "https://github.com/Ajatt-Tools/mpvacious/archive/${version}.tar.gz";
+    sha256 = "1xz4qh2ibfv03m3pfdasim9byvlm78wigx1linmih19vgg99vky2";
```
Comment (SuperSandro2000) [inline pkgs/applications/video/mpv/scripts/mpvacious.nix:8]:
```suggestion
  version = "0.12";

  src = fetchzip {
    url = "https://github.com/Ajatt-Tools/mpvacious/archive/v${version}.tar.gz";
    sha256 = "1xz4qh2ibfv03m3pfdasim9byvlm78wigx1linmih19vgg99vky2";
```

File: pkgs/applications/video/mpv/scripts/mpvacious.nix (hunk 1 above)
Comment (kmicklas) [reply pkgs/applications/video/mpv/scripts/mpvacious.nix]: Ah yeah it's plus. Fixed.

File: pkgs/applications/video/mpv/scripts/mpvacious.nix (hunk 2 above)
Comment (kmicklas) [reply pkgs/applications/video/mpv/scripts/mpvacious.nix]: I originally had it this way but wasn't confident. :smile:

File: pkgs/applications/video/mpv/scripts/mpvacious.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchzip, curl, xclip }:
+
+stdenv.mkDerivation rec {
+  pname = "mpvacious";
+  version = "0.12";
+
+  src = fetchzip {
```
Comment (teto) [inline pkgs/applications/video/mpv/scripts/mpvacious.nix:7]: use fetchFromGitHub instead

File: pkgs/applications/video/mpv/scripts/mpvacious.nix [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  postPatch = ''
+    substituteInPlace subs2srs.lua \
+      --replace "'curl'" "'${curl}/bin/curl'" \
+      --replace "xclip" "${xclip}/bin/xclip"
+  '';
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/share/mpv/scripts
```
Comment (teto) [inline pkgs/applications/video/mpv/scripts/mpvacious.nix:21]: don't forget to run the hooks `runHook preInstall` / postInstall

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 109545` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mpvScripts.mpvacious</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109545` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>mpvScripts.mpvacious</li>
</ul>
</details>

**Pull request merged**